Resources/Views/eventCard.leaf

```diff
@@ -1,5 +1,12 @@
 <a href="/event/#(event.shortname)">
-    <div class="conference-cell-no-image">
+    <div class="event-cell">
         <p>#(event.fullname)</p>
+        #if(event.startDate != nil || event.endDate != nil) {
+            <span class="date">
+                #date(event.startDate, "dd MMM yyyy") - #date(event.endDate, "dd MMM yyyy")
+            </span>
+        } else {
+            <span class="date"></span>
```

Comment:
we don't use the leading spaces after `if` and `else` in the leaf files for now. It is better to have the same style in all codebase if possible 👍

Reply:
fixed.